content/en/docs/cloud-api-adaptor/azure/_index.md

```diff
@@ -220,13 +208,35 @@ export AZURE_SUBNET_ID=$(az network vnet subnet list \
   --output tsv)
 ```
 
+### CAA pod-VM image
+
+Export this environment variable to use for the peer pod VM:
+
+```bash
+export AZURE_IMAGE_ID="/CommunityGalleries/cococommunity-42d8482d-92cd-415b-b332-7648bd978eff/Images/peerpod-podvm-ubuntu2204-cvm-snp/Versions/${CAA_VERSION}"
+```
+
+> **Note**:
+>
+> - If you want to use the nightly build of pod-VM image then follow instructions [here](azure-advanced#use-caa-nightly-pod-vm-image).
```

Comment:
```Suggestion
- If you want to use the nightly build of pod VM image then follow the instructions [here](azure-advanced#use-caa-nightly-pod-vm-image).
```

Reply:
I kept those anchors on purpose because this is how it looks like with them on:

<img width="747" alt="image" src="https://github.com/confidential-containers/confidentialcontainers.org/assets/5815795/b8aaa641-8f81-433b-92d0-55be90c4c956">

And without them it looks like this:

<img width="747" alt="image" src="https://github.com/confidential-containers/confidentialcontainers.org/assets/5815795/259b2b45-1908-4989-9033-4d9dcb5ca7fd">